tests/config/vitest.config.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Vitest config file
+ * @see https://vitest.dev/config/
+ */
+
+import { defineConfig } from 'vitest/config';
+
+const browser = false;
+
+export default defineConfig({
+	test: {
+		environment: 'jsdom',
+		environmentOptions: {
+			jsdom: {
+				console: true,
+				resources: 'usable'
+			}
+		},
+		include: ['tests/unit/**/*.test.ts'],
+		setupFiles: ['tests/config/vitest.setup.ts'],
+		testTimeout: 1000,
+		browser: browser ? {
```

Comment:
Do we want to keep the `browser` setting here for future reference? If not, I'd remove it

Reply:
Right, it's easy to restore in case we ever need.